content/docs/command-reference/update.md

```diff
@@ -46,14 +46,14 @@ $ dvc update --rev master
   > revision.
 
 - `-R`, `--recursive` - determines the files to update by searching each target
-  directory and its subdirectories for import stage `.dvc` files to inspect. If
-  there are no directories among the targets, this option has no effect.
-
-- `--to-remote` - update a `.dvc` file created with `dvc import-url` and
-  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
-  the new data directly to remote storage (the default one unless `-r` is used).
-  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
-  data locally. This option can't be used with DVC or Git repository imports.
+  directory and its subdirectories for import `.dvc` files to inspect. If there
+  are no directories among the targets, this option has no effect.
+
+- `--to-remote` - update a `.dvc` file created with `dvc import-url` without
+  downloading the latest data.
+  [Transfer it](/doc/command-reference/import-url#example-transfer-to-remote-storage)
+  directly to remote storage instead (the default one unless one is specified
+  with `-r`). Use `dvc pull` to get the data locally.
```

Comment:
Made all `--to-remote` descriptions consistent and as simple as possible, including in `update` now.

Reply:
p.s. @dberenbaum I see you started reviewing just before I made this summary 😅 

FYI the only newer commit is the one related to this comment, specifically https://github.com/iterative/dvc.org/pull/3210/commits/bf1cde2b5d0d5da37e7059f1fbe978bf8e51d28a.